writing/privacy.md

```diff
@@ -0,0 +1,21 @@
+# Give up all your data
+
+Imagine a world where every advertisement you see is only something that you want to buy and can afford. If you don't want to buy anything, you don't get any ads. When something you don't know about but would love is released, you got told about it. No more spam! Sounds great!
+
```

Comment:
'get' told about it instead of 'got'
I like what you did with the intro paragraph.